x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -1027,7 +1116,7 @@ var testCases = []struct {
 				switch r.URL.Path {
 				case "/":
 					fmt.Fprintf(w, `{"files":[{"id":"1"},{"id":"2"}],"exportId":"2212", "createdAt":"22/02/2022",
-						"nextLink":"%s/link1"}`, serverURL)
+					"nextLink":"%s/link1"}`, serverURL)
```

Comment:
Please reverse this whitespace change as well. also at L1198.

Reply:
Done.